test/rules/std_test.py

```diff
@@ -31,7 +31,7 @@
         # Check we get comma (with leading space/newline) whitespace errors
         # NB The newline before the comma, should report on the comma, not the newline for clarity.
         ("L005", "whitespace_errors.sql", [(2, 9)]),
-        ("L019", "whitespace_errors.sql", [(4, 1)]),
+        ("L019", "whitespace_errors.sql", {(2, 12), (4, 1)}),
```

Comment:
I'm not sure why the behavior changed here, but if you look at `whitespace_errors.sql`, the new behavior seems correct (unless I'm missing something).